### PR TITLE
Add publications since ISAv8

### PR DESCRIPTION
**Pull request by brooksdavis**

### CODE REVIEW

**Review by bsdjhb — APPROVED** [review]

File: chap-intro.tex [hunk 1]
```diff
@@ -966,6 +966,23 @@ \section{Publications}
...
+  of malicious memory access from pluggable computer peripherals and
+  microcontrollers embedded within a system-on-chip.
+
+\item In Workshop on Computer Architecture Research with RISC-V (CARRV 2021),
+  we published \citetitleit{fuchs2021framework}~\cite{fuchs2021framework}.
+  This paper presents a flexible and extensible bare-metal test suite containing replications of all major transient-execution attacks in RISC-V.
+
+\item In the proceedings of the 31st European Symposium on Programming
+  (ESOP 2022) we published \citetitleit{DBLP:conf/esop/BauereissCSAESB22}~\cite{DBLP:conf/esop/BauereissCSAESB22}.
+  This paper we define the fundamental security property that Morello aims
```
Comment (bsdjhb) [inline chap-intro.tex:982]: Hmm, "In this paper we define..."?

File: chap-intro.tex [hunk 2]
```diff
@@ -966,6 +966,23 @@ \section{Publications}
...
+  to provide, reachable capability monotonicity, and prove that the
```
Comment (bsdjhb) [inline chap-intro.tex:983]:
```suggestion
  to provide: reachable capability monotonicity, and prove that the
```

File: chap-intro.tex [hunk 3]
```diff
@@ -1055,6 +1076,34 @@ \section{Publications}
...
+
+\item Brett Gutstein's PhD dissertation, \citetitleit{UCAM-CL-TR-975},
+  describes a memory-operations framework for reasoning about memory-safety
+  mitigations, presents a CHERI-aware implemention of Apple’s JavaScriptCore,
+  and describes the Cornucopia temporal memory safety implementation~\cite{UCAM-CL-TR-975}.
+
+\item Hesham Almatary's PhD dissertation, \citetitleit{UCAM-CL-TR-976},
+  describes CompartOS a new lightweight hardware-software compartmentalisation
+  model building on CHERI to secure mainstream and complex embedded software
+  systems\cite{UCAM-CL-TR-976}.
```
Comment (bsdjhb) [inline chap-intro.tex:1101]:
```suggestion
  systems~\cite{UCAM-CL-TR-976}.
```

File: chap-intro.tex [hunk 4]
```diff
@@ -1055,6 +1076,34 @@ \section{Publications}
...
+  and cryptographic network tokens to implement object capabilities in a
+  distributed cyber physical system~\cite{UCAM-CL-TR-963}.
+
+\item Brett Gutstein's PhD dissertation, \citetitleit{UCAM-CL-TR-975},
+  describes a memory-operations framework for reasoning about memory-safety
+  mitigations, presents a CHERI-aware implemention of Apple’s JavaScriptCore,
+  and describes the Cornucopia temporal memory safety implementation~\cite{UCAM-CL-TR-975}.
+
+\item Hesham Almatary's PhD dissertation, \citetitleit{UCAM-CL-TR-976},
+  describes CompartOS a new lightweight hardware-software compartmentalisation
```
Comment (bsdjhb) [inline chap-intro.tex:1099]:
```suggestion
  describes CompartOS, a new lightweight hardware-software compartmentalisation
```

File: chap-intro.tex [hunk 5]
```diff
@@ -1055,6 +1076,34 @@ \section{Publications}
...
+  model building on CHERI to secure mainstream and complex embedded software
+  systems\cite{UCAM-CL-TR-976}.
+
+\item Peter Rugg's PhD dissertation, \citetitleit{UCAM-CL-TR-984},
+  describes the implemention of CHERI on three RISC-V microarchitectures
```
Comment (bsdjhb) [inline chap-intro.tex:1104]:
```suggestion
  describes the implementation of CHERI on three RISC-V microarchitectures
```

File: .gitignore [hunk 6]
```diff
@@ -23,3 +23,5 @@
 /sail-cheri-mips
 
 /cheri_concentrate_listings/cheri-cap-lib
+
+/.DS_Store
```
Comment (jrtc27) [inline .gitignore:27]: (Copying from https://github.com/CTSRD-CHERI/cheri-specification/commit/d659b2902cbbf2cd7e523ed86552a2551cb2d835#r123866209 as it doesn't show here)

This belongs in your system's global .gitignore. Repositories' .gitignore files are for things specific to the repository.

File: chap-intro.tex (hunk 2 above)
Comment (bsdjhb) [reply chap-intro.tex:983]: Actually, nm.

File: .gitignore (hunk 6 above)
Comment (brooksdavis) [reply .gitignore]: dropped this change

**Review by bsdjhb — APPROVED** [review]

**Pull request merged**